### PR TITLE
Lucas Colwell

### PR DESCRIPTION
**Pull request by lcole37**

### CODE REVIEW

File: lib/museum.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  end
+
+  def recommend_exhibits(patron)
+    interests = patron.interests
+    recommended = []
+    interests.each do |interest|
+      # binding.pry
+       @exhibits.each do |exhibit|
+        if exhibit.name == interest
+          recommended << exhibit #why is this returning a string? RIP 45 minutes
```
Comment (cjsim89) [inline lib/museum.rb:22]: Not sure, but it looks like you figured it out! Overall good job with nested iteration, starting with `each` is a good idea. Try to come back to this and refactor it using other enumerables, if you have time, for some good practice.

File: spec/exhibit_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+require 'rspec'
+require './lib/exhibit'
+
+RSpec.describe Exhibit do
+  before :each do
```
Comment (cjsim89) [inline spec/exhibit_spec.rb:5]: Good job using `before :each do` - do you find that this makes testing go a little faster during an IC?

File: spec/museum_spec.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    @patron_1.add_interest("Dead Sea Scrolls")
+    @patron_1.add_interest("Gems and Minerals")
+    @patron_2.add_interest("IMAX")
+    @dmns.add_exhibit(@gems_and_minerals)
+    @dmns.add_exhibit(@dead_sea_scrolls)
+    @dmns.add_exhibit(@imax)
+
+    expect(@dmns.recommend_exhibits(@patron_1)).to eq([@dead_sea_scrolls, @gems_and_minerals])
+    expect(@dmns.recommend_exhibits(@patron_2)).to eq([@imax])
+  end
```
Comment (cjsim89) [inline spec/museum_spec.rb:43]: Your testing skills seem solid, good work

**Review by cjsim89 — COMMENTED** [review]
Hey Lucas, good work on this week's IC. When you have time, try to come back and finish out iteration 3 and 4 for good practice! I think your committing, testing, and overall work is solid, just need to see more completed features for the final. Keep up the good work overall!

File: spec/exhibit_spec.rb (hunk 2 above)
Comment (lcole37) [reply spec/exhibit_spec.rb:5]: yes! however, it caused me to forget a couple lines in my test and resulted in that 45 minute hurdle. I am more careful about it now

**Pull request closed without merging**